cmd-export-messages.c

```diff
@@ -229,21 +250,39 @@ text_write_time_field(FILE *fp, const char *field, int64_t msec)
 	    labs(tm->tm_gmtoff) % 3600 / 60);
 }
 
+static char
+*get_attachment_field(struct sbk_attachment *att, int flags)
+{
+	char *att_text=	NULL, *att_fname;
+
+	att_fname=	get_file_name(att, flags);
+	if (att_fname==NULL)
+		att_fname=	strdup("no filename");
+
+	asprintf(&att_text, "%s (%s, %" PRIu64 " bytes, id %s)", att_fname,
+		(att->content_type != NULL) ?
+		att->content_type : "",
+		att->size,
+		sbk_attachment_id_to_string(att));
```

Comment:
Ok, if FLAG_FILENAME_ID is given, we get the ID twice (one in the filename and one here).
I think I should change this to not repeat it here when it is already in the filename.
Do you think so, too?